content/doc/book/pipeline/syntax.adoc

```diff
@@ -1066,7 +1066,7 @@ pipeline {
 The `input` directive on a `stage` allows you to prompt for input, using the
 link:https://jenkins.io/doc/pipeline/steps/pipeline-input-step/#input-wait-for-interactive-input[`input` step].
 The `stage` will pause after any `options` have been applied, and before
-entering the `stage`s `agent` or evaluating its `when` condition. If the `input`
+entering the ``stage``s `agent` or evaluating its `when` condition. If the `input`
```

Comment:
I believe it is trying to use a possessive of stage, showing that the agent is owned by the stage.  The phrasing for the possessive seems odd to me, while using a phrase 'for that `stage`' seems a little clearer.
```suggestion
entering the `agent` block for that `stage` or evaluating the `when` condition of the `stage`. If the `input`
```

Reply:
I agree.